qa/tasks/s3tests.py

```diff
@@ -98,156 +98,93 @@ def create_users(ctx, config):
-                            '-n', client_with_id,
-                            'user', 'create',
-                            '--uid', s3tests_conf[section]['user_id'],
-                            '--display-name', s3tests_conf[section]['display_name'],
-                            '--access-key', s3tests_conf[section]['access_key'],
-                            '--secret', s3tests_conf[section]['secret_key'],
-                            '--email', s3tests_conf[section]['email'],
-                            '--caps', 'user-policy=*',
-                            '--cluster', cluster_name,
-                        ],
-                    )
-                if not ctx.dbstore_variable:
-                    ctx.cluster.only(client).run(
-                            args=[
-                                'adjust-ulimits',
-                                'ceph-coverage',
-                                '{tdir}/archive/coverage'.format(tdir=testdir),
-                                'radosgw-admin',
-                                '-n', client_with_id,
-                                'mfa', 'create',
-                                '--uid', s3tests_conf[section]['user_id'],
-                                '--totp-serial', s3tests_conf[section]['totp_serial'],
-                                '--totp-seed', s3tests_conf[section]['totp_seed'],
-                                '--totp-seconds', s3tests_conf[section]['totp_seconds'],
-                                '--totp-window', '8',
-                                '--totp-seed-type', 'base32',
-                                '--cluster', cluster_name,
-                            ],
-                        )
+                    args=[
+                        'adjust-ulimits',
+                        'ceph-coverage',
+                        '{tdir}/archive/coverage'.format(tdir=testdir),
+                        'radosgw-admin',
+                        '-n', client_with_id,
+                        'mfa', 'create',
+                        '--uid', s3tests_conf[section]['user_id'],
+                        '--totp-serial', s3tests_conf[section]['totp_serial'],
+                        '--totp-seed', s3tests_conf[section]['totp_seed'],
+                        '--totp-seconds', s3tests_conf[section]['totp_seconds'],
+                        '--totp-window', '8',
+                        '--totp-seed-type', 'base32',
+                        '--cluster', cluster_name,
+                    ],
+                )
+
+            # add/configure caps for iam user
+            if section=='iam':
+                ctx.cluster.only(client).run(
+                    args=[
+                        'adjust-ulimits',
+                        'ceph-coverage',
+                        '{tdir}/archive/coverage'.format(tdir=testdir),
+                        'radosgw-admin',
+                        '-n', client_with_id,
+                        'caps', 'add',
```

Comment:
I am not sure if this command shall work with dbstore yet. Do you have results of dbstore suite?